shared/lib/constants.py

```diff
@@ -459,3 +459,10 @@
 
 # English language code.
 EN_LANG_CODE = 'en'
+
+# date query param value for requesting latest observations from REST v2 API
+DATE_LATEST = 'LATEST'
+
+# date query param value for requesting latest overall available for
```

Comment:
nit: note that the difference between DATE_LATEST and DATE_HIGHEST_COVERAGE is that one will return the same date for all the data and the other could have different dates?

Reply:
Done